.expeditor/verify.pipeline.yml

```diff
@@ -6,45 +6,73 @@ expeditor:
 
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+- label: run-lint-ruby-2.4
+  command:
+    - apt-get update
+    - apt-get install -y libarchive-dev
+    - .expeditor/run_linux_tests.sh rake style
+  expeditor:
+    executor:
+      docker:
+        image: ruby:2.4
+
+- label: run-specs-ruby-2.4
   command:
     - apt-get update
     - apt-get install -y libarchive-dev
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.4-buster
+        image: ruby:2.4
 
-- label: run-lint-and-specs-ruby-2.5
+- label: run-specs-ruby-2.5
   command:
     - apt-get update
     - apt-get install -y libarchive-dev
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.5-buster
+        image: ruby:2.5
 
-- label: run-lint-and-specs-ruby-2.6
+- label: run-specs-ruby-2.6
   command:
     - apt-get update
     - apt-get install -y libarchive-dev
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.6-buster
+        image: ruby:2.6
 
-- label: run-lint-and-specs-ruby-2.7
+- label: run-specs-ruby-2.7
   command:
     - apt-get update
     - apt-get install -y libarchive-dev
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.7-buster
+        image: ruby:2.7
+
+- label: run-specs-ruby-3.0
+  command:
+    - apt-get update
```

Comment:
Please remove `apt-get update` for ruby 3.0 and 3.1 tests since you aren't running `apt-get install`.